packages/pwa/app/utils/url.js

```diff
@@ -79,19 +79,22 @@ export const buildUrlSet = (url = '', key = '', values = [], extraParams = {}) =
  *
  * @param {Object} category
  * @param {string} locale
+ * @param {string} siteAlias
  * @returns {string}
  */
-export const categoryUrlBuilder = (category, locale = DEFAULT_LOCALE) =>
-    encodeURI(`/${locale}/category/${category.id}`)
+export const categoryUrlBuilder = (category, locale = DEFAULT_LOCALE, siteAlias) =>
+    encodeURI(`/${siteAlias}/${locale}/category/${category.id}`)
 
 /**
  * Given a product and the current locale returns an href to the product detail page.
  *
  * @param {Object} product
  * @param {string} locale
+ * @param {string} siteAlias
  * @returns {string}
  */
-export const productUrlBuilder = (product) => encodeURI(`/product/${product.id}`)
+export const productUrlBuilder = (product, locale, siteAlias) =>
```

Comment:
To make this function extendable in the future, I think this function's parameters could use an object. Something like this: `(product, {locale, siteAlias})` OR `(product, options)`.

Product info is required, but the rest can be optional. Some sites may enable locale/site alias or not. And in the future, we can add more things to the object `options`.